Bottom: 5636325b55a34647e2f5b9761b985fc3548952e3
Top:    ab919816fcfa9d960b2938216bcf15242ae9e8a5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-23 01:09:24 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
